nodejs/eks/cni/aws-k8s-cni.yaml

```diff
@@ -1,224 +1,576 @@
-              openAPIV3Schema:
-                  type: object
-                  x-kubernetes-preserve-unknown-fields: true
-    names:
-        plural: eniconfigs
-        singular: eniconfig
-        kind: ENIConfig
+    spec:
+      priorityClassName: "system-node-critical"
+      serviceAccountName: aws-node
+      hostNetwork: true
+      initContainers:
+      - name: aws-vpc-cni-init
+        image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni-init:v1.16.0
+        env: []
+        # env:
+          # - name: DISABLE_TCP_EARLY_DEMUX
+          #   value: "false"
+          # - name: ENABLE_IPv6
+          #   value: "false"
+        securityContext:
+            privileged: true
+        resources:
+            requests:
+              cpu: 25m
+        volumeMounts:
+          - mountPath: /host/opt/cni/bin
+            name: cni-bin-dir
+      terminationGracePeriodSeconds: 10
+      tolerations:
+        - operator: Exists
+      securityContext:
+        {}
+      containers:
+        - name: aws-node
+          image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.16.0
+          ports:
+            - containerPort: 61678
+              name: metrics
+          livenessProbe:
+            exec:
+              command:
+              - /app/grpc-health-probe
+              - -addr=:50051
+              - -connect-timeout=5s
+              - -rpc-timeout=5s
+            initialDelaySeconds: 60
+            timeoutSeconds: 10
+          readinessProbe:
+            exec:
+              command:
+              - /app/grpc-health-probe
+              - -addr=:50051
+              - -connect-timeout=5s
+              - -rpc-timeout=5s
+            initialDelaySeconds: 1
+            timeoutSeconds: 10
+          env:
+            - name: ADDITIONAL_ENI_TAGS
+              value: "{}"
+            # - name: AWS_VPC_CNI_NODE_PORT_SUPPORT
+            #   value: "true"
+            # - name: AWS_VPC_ENI_MTU
+            #   value: "9001"
+            # - name: AWS_VPC_K8S_CNI_CUSTOM_NETWORK_CFG
+              # value: "false"
+            # - name: AWS_VPC_K8S_CNI_EXTERNALSNAT
+            #   value: "false"
+            # - name: AWS_VPC_K8S_CNI_LOGLEVEL
+            #   value: "DEBUG"
+            # - name: AWS_VPC_K8S_CNI_LOG_FILE
+            #   value: "/host/var/log/aws-routed-eni/ipamd.log"
+            - name: AWS_VPC_K8S_CNI_RANDOMIZESNAT
+              value: "prng"
+            # - name: AWS_VPC_K8S_CNI_VETHPREFIX
+            #   value: "eni"
+            # - name: AWS_VPC_K8S_PLUGIN_LOG_FILE
+            #   value: "/var/log/aws-routed-eni/plugin.log"
+            # - name: AWS_VPC_K8S_PLUGIN_LOG_LEVEL
+            #   value: "DEBUG"
+            - name: DISABLE_INTROSPECTION
+              value: "false"
+            - name: DISABLE_METRICS
+              value: "false"
+            - name: DISABLE_NETWORK_RESOURCE_PROVISIONING
+              value: "false"
+            - name: ENABLE_IPv4
```

Comment:
fyi: this was removed in the previous version of the CNI manifest used in pulumi-eks, however, we need this value to be explicitly enabled otherwise the `aws-node` container will fail on startup resulting in a crashloop backoff.